src/Containers/Rockets/Card/Reservation/ReservationBtn.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { bookRocket, cancelBooking } from '../../../../redux/rockets';
```

Comment:
- [ ] **[OPTIONAL]** Be careful not to nest a lot of folders, so you have better imports, this ReservationBtn component can be moved to our "Components" folder and be imported from there.